### DISCUSSION TITLE
Welcome to dotnes ".NES" Discussions!

### DISCUSSION THREAD

**Discussion by jonathanpeppers** [jonathanpeppers/dotnes#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community interested in retro gaming, .NET, etc. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I'm @jonathanpeppers! I work for Microsoft on .NET for Android and .NET MAUI.

I have always had an interest in gaming, but I was lead developer on some popular mobile apps:

* [Draw a Stickman: EPIC](https://drawastickman.com/epic1) and [EPIC 2](https://drawastickman.com/epic2)
* [Hanx Writer](http://hanxwriter.com/) (built for Tom Hanks)

**Comment by longjoel** [comment]
Is anybody working on a port of neslib for .net? It might be nice to have a reference implementation, and faster dev times.

**Reply by jonathanpeppers** [reply]
There might be a few features that would be required to do something like that, for example:
* C# static methods -> convert to subroutines
